protocol-definitions/Sql.yaml

```diff
@@ -222,6 +222,12 @@ methods:
           since: 2.5
           doc: |
             Is the result set unbounded.
+        - name: partitionArgumentIndex
+          type: int
+          nullable: false
+          since: 2.5
```

Comment:
we had a [discussion](https://hazelcast.slack.com/archives/C034YQBTG/p1668784483838309) about using Hazelcast Version for `since` but without final conclusion yet. Anyway, I think it shoud not be 2.5 which is version for 5.2